File: fizzbuzz.js
/* Directions
Write a program that console logs the numbers from 1 to n.
For multiples of three print 'fizz' instead of the number
and for multiples of five print 'buzz'.  For numbers which are
multiples of both three and five print 'fizzbuzz'.
*/
//The real challenge of this is determining when the number is
//a multiple of both three and five - modulo operator helps this!

function fizzBuzz(n) {
    for (let i = 1; i<=n; i++) {
        //multiple of 3 & 5 (should only be multiples of 15)
        //could do (i % 15 === 0)
        if (i % 3 === 0 && i % 5 == 0) {
            console.log('fizzbuzz');
        }
        else if (i % 3 === 0) {
            console.log('fizz');
        }
        else if (i % 5 === 0) {
            console.log('buzz');
        }
        else {
            console.log(i);
        }
    }
}

module.exports = fizzBuzz;

File: Python challenges/fizzbuzz.py
#Write a program that prints integers 1 to 100. For multiples
# of three print "Fizz" instead of the integer.  For multiples
# of five print "Buzz".  For integers which are multiples of
# both three and five print "FizzBuzz".
#

for num in range(1,101):
    if num%3 == 0 and num%5 == 0:
        print("FizzBuzz")
    elif num%3 == 0:
        print("Fizz")
    elif num%5 == 0:
        print("Buzz")
    else:
        print(num)